README.md

```diff
@@ -11,33 +11,33 @@ have the mapping run purely client side.
 
 ## For Contributors
 
-To build Blaze Map locally, make sure to have a copy of the Rubidium binary stored under `/libs`.
-This is because Rubidium is a build time dependency, even if only an optional runtime dependency.
+To build Blaze Map locally, make sure to have a copy of the Embeddium binary stored under `/libs`.
+This is because Embeddium is a build time dependency, even if only an optional runtime dependency.
 
-You can see what version of the Rubidium binary is needed by looking at which version is listed 
+You can see what version of the Embeddium binary is needed by looking at which version is listed
 in `build.gradle`.
 
 ### Local Dev
 
-To build and run the local dev server in single player mode (client with integrated server):
```

Comment:
The ordering of the bits under this heading was changed but the content is the same